.gitignore

```diff
@@ -1,25 +1,26 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
+/*.py[cod]
+/src/PIL/*.py[cod]
+/Tests/*.py[cod]
 
 # C extensions
-*.so
+/src/PIL/*.so
 
 # Distribution / packaging
+/build/
+/dist/
+/src/*.egg-info/
+/env/
 .eggs/
```

Comment:
To be honest, I'd rather remove all rules stating from this until the end of the section. I haven't ever seen folders with this names. If someone believes that something should be left, please leave comments on the corresponding line.

Reply:
`build` is used by setuptools as an intermediate build directory (often not cleaned properly), and `dist` is where it puts wheels/eggs.

`env` is probably for virtual environments, and likely unnecessary (one can just put a `*` gitignore in the subdirectory if they need it).

I would guess the other directories are for older versions of setuptools/distutils.